packages/shared/src/components/profile/ProfileSettingsMenu.tsx

```diff
@@ -203,6 +203,16 @@ export const accountPageItems = defineMenuItems({
       },
     },
   },
+  logout: {
+    title: null,
```

Comment:
What does it mean if the title here is `null`?

Reply:
It's just the section title. When it is null, it doesn't render a section title like for the others
![image](https://github.com/user-attachments/assets/4dc1b5f4-910c-462e-92c7-aee8a374ccec)

Reply:
Ah, cool. Thanks for letting me know 🙌

Reply:
And TypeScript being TypeScript, it had to be null, otherwise it throws this type error
<img width="1158" alt="image" src="https://github.com/user-attachments/assets/fcae3fb4-e22e-4048-8ca6-599a18ab61aa" />